internal/uvm/create_lcow.go

```diff
@@ -110,6 +111,7 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		VPMemDeviceCount:      DefaultVPMEMCount,
 		VPMemSizeBytes:        DefaultVPMemSizeBytes,
 		PreferredRootFSType:   PreferredRootFSTypeInitRd,
+		EnableColdDiscardHint: false,
```

Comment:
Ok last thing :). This is the `default` value of a `bool` type so we dont need it here. Otherwise LGTM

Reply:
This follows the style of the other fields on OptionsLCOW, such as EnableGraphicsConsole which also sets its default as false.

Reply:
I just dont know why we would do that but ok fine :).